docs/docs/50-user-guide/60-reference-docs/30-promotion-steps/jira.md

```diff
@@ -0,0 +1,651 @@
+---
+sidebar_label: jira
+description: Integrates with Jira to manage issues, comments, and track promotion workflows.
+---
+
+# `jira`
+
+:::info Enterprise Feature
+This promotion step is only available in Akuity Promote, you can use this feature in Kargo v1.6+ offered on [Akuity Platform](https://akuity.io/akuity-platform).
+:::
```

Comment:
@kencochrane this is a spot we were talking about offline where we need to nail down wording to be more clear.